### PR TITLE
Fixes #35, wraps select and arrows in div, position relative.

### LINKED ISSUE #35: Select box caret icon misplaced
Hey guys, thought you might wanna know. When validation on a select input fails, the caret icon shifts down (see the image)

screen shot 2015-08-24 at 8 36 26 pm

Cheers!

### PR DESCRIPTION
**Pull request by NogsMPLS**

### CODE REVIEW

File: src/components/FormSelect.js [hunk 1]
```diff
@@ -123,10 +123,12 @@ module.exports = React.createClass({
 		return (
 			<div className={componentClass}>
 				{componentLabel}
-				<select className="FormInput FormSelect" id={forAndID} onChange={this.handleChange} onBlur={this.handleBlur} {...props}>
-					{options}
-				</select>
-				{this.renderIcon(icons.selectArrows)}
+				<div className="u-pos-relative">
```
Comment (nkbt) [inline src/components/FormSelect.js]: Select is inline element, wrapping it in a block element should be a trouble. Maybe span?

**Comment by NogsMPLS** [comment]
I think it needs to be a block element because of the way validation is inserting messages.

What's causing this bug is the validation message being appended to the main div ```<div className={componentClass}>``` and adding height. Because the arrows have ```bottom: 0``` they cling to the bottom of the parent div.

By wrapping select element and renderIcon in their own div the arrows from renderIcon will ignore the added height to the outer div. Using a span with position relative does not fix this.

Is there a use case where this would mess something up that I'm unaware of?

**Comment by jossmac** [comment]
@NogsMPLS  thanks for this

@nkbt we declare `width: 100%` via `.FormInput` on all input components so this should be fine

**Comment by nkbt** [comment]
I see... Almost forgot that there is still some sort of global css around.
Too used to css-modules :).

**Comment by jossmac** [comment]
@nkbt you're living the dream :)

**Pull request merged**